protocol/app/upgrades_test.go

```diff
@@ -1,17 +1,10 @@
 package app_test
 
-import (
-	"errors"
-	"testing"
+/*
```

Comment:
should we just remove these tests? these should already be covered in `x/upgrade` so I don't think these tests are too useful

Reply:
yup I agree